content/docs/reference/version-policy.md

```diff
@@ -0,0 +1,262 @@
+documentation for that application.
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Application</th>
+        <th>Status in Kubeflow {{% kf-latest-version %}}</th>
+        <th>Application version in Kubeflow {{% kf-latest-version %}}</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="/docs/components/central-dash/overview/">Central 
+          dashboard: Kubeflow UI</a>
+          (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/centraldashboard">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/chainer/">Chainer operator</a>
+        (<a href="https://github.com/kubeflow/chainer-operator">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/hyperparameter-tuning/overview/">Hyperparameter
+          tuning: Katib</a>
+          (<a href="https://github.com/kubeflow/katib">GitHub</a>)
+          </td>
+        <td>Beta</td>
+        <td>v1alpha3</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/serving/kfserving/">KFServing</a>
+          (<a href="https://github.com/kubeflow/kfserving">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td>v0.2.2</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/misc/metadata/">Metadata</a>
+          (<a href="https://github.com/kubeflow/metadata">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/mpi/">MPI training: MPI 
+          operator</a>
+          (<a href="https://github.com/kubeflow/mpi-operator">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/mxnet/">MXNet training: MXNet 
+          operator</a>
+          (<a href="https://github.com/kubeflow/mxnet-operator">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/notebooks/why-use-jupyter-notebook/">Notebook web
+          app</a>
+          (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/jupyter-web-app">GitHub</a>)
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/notebooks/why-use-jupyter-notebook/">Notebook 
+          controller</a> 
+          (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/notebook-controller">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/pipelines/overview/pipelines-overview/">Pipelines</a>
+          (<a href="https://github.com/kubeflow/pipelines">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td>0.1.40</td>
```

Comment:
/cc @IronPan please confirm the current release version of Pipelines currently deployed with Kubeflow dpeloyments. If there is any plan to update to a more later release version, please update here.

Reply:
We are shipping pipelines 0.2.0
https://github.com/kubeflow/manifests/blob/v1.0-branch/pipeline/pipelines-ui/base/kustomization.yaml#L15

Reply:
thanks @jlewi. @sarahmaddox please update to 0.2.0 for pipelines.

Reply:
Done.